### PR TITLE
[Issue #163] Publish OGC test results 

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,12 @@ env:
   matrix:
   - TESTS=OGC_TESTS
   - TESTS=API
+  global:
+  - SSH_KEY="id_rsa"
+  - GIT_NAME="Isabel Rios"
+  - GIT_EMAIL="isabelrios@gmail.com"
+  - SOURCE_DIR="/home/travis/build/mozilla-sensorweb/sensorthings/test/TE_BASE/users/travis/s0001/html/sta10-0.8-SNAPSHOT.html"
```
Comment (ferjm) [inline .travis.yml]: You may want to publish the entire `html` dir

Comment (isabelrios) [inline .travis.yml]: Yes, that will be better.

I am having troubles as travis says that:

> SOURCE_DIR (/home/travis/build/mozilla-sensorweb/sensorthings/test/TE_BASE/users/travis/s0001/html/sta10-0.8-SNAPSHOT.html) does not exist, build the source directory before deploying

So I am creating it after launching the tests to be sure the dir is already created.
Not sure if that will work or if there is a better solution

**Comment by isabelrios** [comment]
@ferjm after having travis green, r? please.
Thanks!

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,20 @@ jdk:
...
   matrix:
-   - TESTS=OGC_TESTS
-   - TESTS=API
+  - TESTS=OGC_TESTS
+  - TESTS=API
+  global:
+  - SSH_KEY="id_rsa"
+  - GIT_NAME="Isabel Rios"
+  - GIT_EMAIL="isabelrios@gmail.com"
+  # - SOURCE_DIR="/home/travis/build/mozilla-sensorweb/sensorthings/test/TE_BASE/users/travis/s0001/html/sta10-0.8-SNAPSHOT.html"
```
Comment (ferjm) [inline .travis.yml]: I guess we need to remove this line.

File: .travis.yml [hunk 3]
```diff
@@ -17,13 +25,17 @@ before_install:
...
 script:
-- if [[ "$TESTS" = "OGC_TESTS" ]]; then source ./scripts/OGC_tests/install_tests.sh ; fi
+- if [[ "$TESTS" = "OGC_TESTS" ]]; then source ./scripts/OGC_tests/install_tests.sh
+  ; fi
 - if [[ "$TESTS" = "OGC_TESTS" ]]; then source ./scripts/preconditions.sh ; fi
-- if [[ "$TESTS" = "OGC_TESTS" ]]; then source ./scripts/OGC_tests/launch_tests.sh ; fi
+- if [[ "$TESTS" = "OGC_TESTS" ]]; then source ./scripts/OGC_tests/launch_tests.sh
+  ; fi
+after_script:
+- if [[ "$TESTS" = "OGC_TESTS" ]]; then npm run gh-pages-travis; fi
```
Comment (ferjm) [inline .travis.yml]: We need to run `check_tests_failures.sh` after `gh-pages-travis`, right?

File: scripts/OGC_tests/launch_tests.sh [hunk 4]
```diff
@@ -10,5 +10,9 @@ echo "Launching SensorThings Test Suite tests"
...
 $COMMON_PATH/test/TE_BASE/te-install/bin/unix/test.sh -source=sta10/1.0/ctl/sta10-suite.ctl -form=$COMMON_PATH/test/forms/sta10.xml
 
+export SOURCE_DIR=$BASE_PATH/test/TE_BASE/users/travis/s0001/html/
+export PATH=$SOURCE_DIR:$PATH
+
+echo $SOURCE_DIR
 # Check if there has been failures running OGC tests
-source ./scripts/OGC_tests/check_tests_failures.sh
\ No newline at end of file
+# source ./scripts/OGC_tests/check_tests_failures.sh
```
Comment (ferjm) [inline scripts/OGC_tests/launch_tests.sh]: Remove this line, please

File: package.json [hunk 5]
```diff
@@ -15,7 +15,9 @@
     "pretest": "./scripts/pretest.sh",
     "test-nocoverage": "NODE_ENV=test ./node_modules/.bin/mocha --delay --compilers js:babel-register test/test_*.js",
     "test-watch": "nodemon -q -x npm test",
-    "test": "./node_modules/.bin/nyc npm run test-nocoverage && npm run lint"
+    "test": "./node_modules/.bin/nyc npm run test-nocoverage && npm run lint",
+    "gh-pages-travis": "gh-pages-travis",
+    "check-failures": "./scripts/OGC_tests/check_tests_failures.sh"
```
Comment (ferjm) [inline package.json]: No need for this. We can simply call `./scripts/OGC_tests/check_tests_failures.sh` from the `.travis.yml` file like we do with the other scripts.

**Pull request merged**